extensions/notebook/src/extension.ts

```diff
@@ -14,11 +14,15 @@ import { IExtensionApi, IPackageManageProvider } from './types';
 import { CellType } from './contracts/content';
 import { NotebookUriHandler } from './protocol/notebookUriHandler';
 import { BuiltInCommands, unsavedBooksContextKey } from './common/constants';
+import { RemoteBookController } from './book/remoteBookController';
+import { RemoteBookDialog } from './dialog/remoteBookDialog';
+import { RemoteBookDialogModel } from './dialog/remoteBookDialogModel';
 
 const localize = nls.loadMessageBundle();
 
 let controller: JupyterController;
 type ChooseCellType = { label: string, id: CellType };
+let remoteBookController: RemoteBookController;
```

Comment:
You don't need to define this up here since you only use it in the activate function.